### PR TITLE
Submitting studio-cli version 23.9.3

### PR DESCRIPTION
**Pull request by TotTheMax**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135271)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   C6E4A0B81E1DAF13AE6216D8A81432A72567AF295393F0D3D8A604036A8415A6
 Expected hash: cd8529bc782b8cca09b58048a20dd4de994a389540be4d7a2963ad948466cf33

(Automated response - build 615.)

File: manifests/w/WindRiver/studio-cli/23.9.3/WindRiver.studio-cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - RelativeFilePath: glab.exe
+    PortableCommandAlias: glab
+  - RelativeFilePath: mc.exe
+    PortableCommandAlias: mc
+  - RelativeFilePath: vault.exe
+    PortableCommandAlias: vault
+  - RelativeFilePath: gitlab.exe
+    PortableCommandAlias: gitlab
+  InstallerUrl: https://distro.windriver.com/dist/wrstudio/wrstudio-cli-distro-cd-staged/23.9.3/win64/studio-cli_all.zip
+  InstallerSha256: cd8529bc782b8cca09b58048a20dd4de994a389540be4d7a2963ad948466cf33
```
Comment (stephengillie) [inline manifests/w/WindRiver/studio-cli/23.9.3/WindRiver.studio-cli.installer.yaml:23]:
```suggestion
  InstallerSha256: C6E4A0B81E1DAF13AE6216D8A81432A72567AF295393F0D3D8A604036A8415A6
```

**Comment by TotTheMax** [comment]
Installer Verification Analysis Context Information: 
Installer https://distro.windriver.com/dist/wrstudio/wrstudio-cli-distro-cd-staged/23.9.3/win64/studio-cli_all.zip
Scan request id: '5e6c2598-5822-471d-b5ea-5218b3c4ce35'
Error information: BlockingDetectionFound

**Comment by TotTheMax** [comment]
Hi, friends. Now and several PRs I have closed before, the error above has always occurred. Does anyone know why?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**